capture/plugins/writer-s3/index.js

```diff
@@ -59,6 +59,14 @@ function makeS3 (node, region) {
     s3Params.secretAccessKey = secret;
   }
 
+  if (Config.getFull(node, 's3Host') !== undefined) {
+    s3Params.endpoint = Config.getFull(node, 's3Host')
+  }
+
+  if (Config.getFull(node, 's3PathAccessStyle') === 'true') {
```

Comment:
So if we want to match the capture side this would need to get the s3Bucket var and set default to true if there is a period

   s3PathAccessStyle     = moloch_config_boolean(NULL, "s3PathAccessStyle", strchr(s3Bucket, '.') != NULL);

Reply:
Added a getBool wrapper and updated this to match the C code.

Reply:
thanks